SignalMessaging/environment/AppSetup.m

```diff
@@ -125,6 +125,10 @@ + (void)setupEnvironmentWithAppSpecificSingletonBlock:(dispatch_block_t)appSpeci
                                                                       syncManager:syncManager
                                                                  typingIndicators:typingIndicators
                                                               attachmentDownloads:attachmentDownloads]];
+        
+        // Loki
+        OWSLokiFriendRequestExpireJob *lokiFriendRequestExpireJob = [[OWSLokiFriendRequestExpireJob alloc] initWithPrimaryStorage:primaryStorage];
+        [SSKEnvironment.shared setLokiFriendRequestExpireJob:lokiFriendRequestExpireJob];
```

Comment:
You can just do `SSKEnvironment.shared.lokiFriendRequestExpireJob = lokiFriendRequestExpireJob`, since you're overriding the setter